test/parallel/test-fs-cp.mjs

```diff
@@ -858,10 +860,11 @@ if (!isWindows) {
 
 // It throws if options is not object.
 {
-  assert.throws(
-    () => cp('a', 'b', { mode: -1 }, () => {}),
-    { code: 'ERR_OUT_OF_RANGE' }
-  );
+  const src = './test/fixtures/copy/kitchen-sink';
+  const dest = nextdir();
+  cp(src, dest, mustNotMutateObjectDeep({ recursive: true, mode: -1 }), (err) => {
```

Comment:
The callback must be wrapped in `mustCall()`. But I am really not sure if argument validation errors should be async, it seems very unusual.

Reply:
Hmm...the `THROW_` macros being run _before_ the `AsyncDestCall` means this should actually be thrown synchronously, so it wouldn't get to the callback at all. Seems to me like it should be throwing from the `cp(...)` call itself though, so I'm not sure how it could be getting to the rest of the test if that's the case. 🤔